cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4342,4 +4342,44 @@
       The number of boss threads for the task worker.
     </description>
   </property>
+
+  <property>
+    <name>artifact.localizer.container.num.cores</name>
+    <value>1</value>
```

Comment:
Do we need a whole core? Or is there a multiplier that will get applied?

Reply:
ResourceSpecification accept int for setVirtualCores. CPU_MULTIPLIER applies to ResourceSpecification of all runnables.